README.md

```diff
@@ -32,12 +32,14 @@ It is also important that your disks are setup according to the [SAP storage req
 
 ## Role Variables
 
-| variable | info | required? |
-|:--------:|:----:|:---------:|
-|sap_hana_deployment_bundle_path|Target host directory path where SAP Installation Bundle SAR file is located|yes|
-|sap_hana_deployment_bundle_sar_file_name|Installation Bundle SAR file name|yes|
-|sap_hana_deployment_sapcar_path|Target host directory directory path where SAPCAR tool file is located|yes|
-|sap_hana_deployment_sapcar_file_name|SAPCAR tool file name|yes|
+| variable |             info             | required? |
+|:--------:|:----------------------------:|:---------:|
+|sap_hana_installdir| directory, where hdblcm is located | No, if `sap_hana_deployment_bundle_path`, `sap_hana_deployment_bundle_sar_file_name`, `sap_hana_deployment_sapcar_path` and `sap_hana_deployment_sapcar_file_name`	are defined.|
```

Comment:
This variable must be defined under `defaults`

Reply:
done